lib/phoenix_live_view/plug.ex

```diff
@@ -50,6 +50,7 @@ defmodule Phoenix.LiveView.Plug do
     for key_or_pair <- session_keys, into: %{} do
       case key_or_pair do
         key when is_atom(key) -> {key, Conn.get_session(conn, key)}
+        key when is_binary(key) -> {key, Conn.get_session(conn, key)}
```

Comment:
@josevalim 🤦‍♂I missed this when I was updating LiveView.Static

Is it safe to `to_string` the atom keys here, or do we also want to signal in the Router that these keys are Strings?